components/addTask.vue

```diff
@@ -3,6 +3,7 @@
     <label for="add task" class="flex-1">
       <input
         type="text"
+        v-model="newTask"
```

Comment:
no .trim on data

Reply:
I thought it wasn't needed for the task data. Will add it now :)